app/src/main/java/com/android/shelfLife/ui/recipes/Recipes.kt

```diff
@@ -2,203 +2,178 @@ package com.android.shelfLife.ui.recipes
-  //
-  //                      if (filteredRecipes.isEmpty()) {
-  //                        Box(
-  //                            modifier = Modifier.fillMaxSize(),
-  //                            content = {
-  //                              Text(
-  //                                  text = "No recipes available",
-  //                                  modifier = Modifier.testTag("noRecipesAvailableText"))
-  //                            },
-  //                            contentAlignment = Alignment.Center)
-  //                      } else {
-  //                        // LazyColumn for displaying the list of filtered recipes
-  //                        LazyColumn(modifier = Modifier.fillMaxSize().testTag("recipesList")) {
-  //                          items(filteredRecipes) { recipe ->
-  //                            RecipeItem(recipe, navigationActions, listRecipesViewModel)
-  //                          }
-  //                        }
-  //                      }
-  //                    }
-  //              })
-  //        }
-  //      }
-}
-/**
- * Converts a string representation of a recipe type into a corresponding
- * `RecipesRepository.SearchRecipeType` enumeration value.
- *
- * @param string A string describing the type of recipe to search for. Possible values include:
- *     - "Soon to expire": Recipes with ingredients that are nearing expiration.
- *     - "Only household items": Recipes using only ingredients available in the household.
- *     - "High protein": Recipes with a high protein content.
- *     - "Low calories": Recipes with low caloric content.
- *
- * @return The corresponding `RecipesRepository.SearchRecipeType` enum value.
- * @throws IllegalArgumentException If the input string does not match any known recipe type.
- */
-fun stringToSearchRecipeType(string: String): RecipeType {
-  return when (string) {
-    "Soon to expire" -> RecipeType.USE_SOON_TO_EXPIRE
-    "Only household items" -> RecipeType.USE_ONLY_HOUSEHOLD_ITEMS
-    "High protein" -> RecipeType.HIGH_PROTEIN
-    "Low calories" -> RecipeType.LOW_CALORIE
-    "Personal" -> RecipeType.PERSONAL
-    else -> throw IllegalArgumentException("Unknown filter: $string")
+fun RecipesScreen(navigationActions: NavigationActions) {
+  val recipesViewModel = hiltViewModel<RecipesViewModel>()
+  val overviewScreenViewModel = hiltViewModel<OverviewScreenViewModel>()
+
+  val user = recipesViewModel.user.collectAsState()
+  val selectedHousehold = recipesViewModel.household.collectAsState()
+
+  val isFabExpanded by recipesViewModel.fabExpanded.collectAsState()
+  val query by recipesViewModel.query.collectAsState()
+  val filteredRecipeList by recipesViewModel.filteredRecipes.collectAsState()
+  val selectedFilters by recipesViewModel.selectedFilters.collectAsState()
+
+  if (selectedHousehold == null) {
+    FirstTimeWelcomeScreen(navigationActions, overviewScreenViewModel) // TODO("kill yourself")
```

Comment:
interesting todo but I'll be ok, also why does this go to the firstTimeWelcomeScreen?

Reply:
it was added originally in the screen at the very beginning . the todo was added by Ricardo and it is there to tell me to use the navigateTo. I will make that change on another branch.